src/BrandwatchReactAuth.js

```diff
@@ -1,20 +1,22 @@
 import PropTypes from 'prop-types';
-import React, { Component } from 'react';
+import { Component } from 'react';
 import jwtDecode from 'jwt-decode';
 import TokenStore from 'donny-auth';
 
 export default class BrandwatchReactAuth extends Component {
   getChildContext() {
     return {
-      brandwatchAuthLogout: this.handleLogout,
       brandwatchAuthGetProfile: this.handleGetProfile,
       brandwatchAuthGetToken: this.handleGetToken,
+      brandwatchAuthHandleError: this.handleError,
```

Comment:
Now seeing what this does, I'm not sure about the name. It's more of a `handleLogout` but with the `backupRedirect` too?

Reply:
Yes good point, logout is a bit odd at the moment, as it only logs you out of the app you are in, really it should remove all tokens. I would be surprised to click logout and be redirected to `my.brandwatch`, as I would then want to log out of that too.